code/controllers/CommentingController.php

```diff
@@ -13,10 +13,27 @@ class CommentingController extends Controller {
 		'approve',
 		'rss',
 		'CommentsForm',
+		'reply',
 		'doPostComment',
 		'doPreviewComment'
 	);
 
+	private static $url_handlers = array(
+		'reply/$ParentCommentID//$ID/$OtherID' => 'reply',
```

Comment:
It is actually :D

That denotes the 'optional' params, plus some other rules in the routing api.